tests/testsuite.src/syn_misc.at

```diff
@@ -9220,3 +9226,44 @@ cobol85.cob:14: warning: optional period used
 ])
 
 AT_CLEANUP
+
+
+AT_SETUP([AREACHECK])
+AT_KEYWORDS([misc])
+
+AT_DATA([pgm1.cob], [
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. PGM1.
+       ENVIRONMENT DIVISION.
+       INPUT-OUTPUT SECTION.
+       FILE-CONTROL.
+       SELECT FILE01 ASSIGN "file01.dat".
+       DATA DIVISION.
+       FILE SECTION.
+       FD FILE01.
+       01 RECORD-FILE01 PIC X(50).
+       WORKING-STORAGE SECTION.
+       01 W-DATA01.
+      * 02 items may still lie in Area A for now:
```

Comment:
The purpose is to adjust this and warn/error there, too; correct?

Reply:
We first need to identify dialects where these non-toplevel numbers *must not* be within Area A. If there are some, then a dedicated dialect option is probably in order to refine those checks.